appyters/RNA_Level_Functional_Predictions/static/AutocompleteField.js

```diff
@@ -0,0 +1,123 @@
+// Adapted from W3schools
+
+function runall(name, file_path){
+    
+function readJSONFromURL(file_path) {
+    $.getJSON(file_path, function(data) {
+        var lst = [];
+        var keys = Object.keys(data);
+        for (var k in keys) {
+            lst = lst.concat(data[keys[k]]);
+        } 
+        autocomplete(document.getElementById(name), lst); 
+    });
+}
+
+function readTextFile(file, callback) {
```

Comment:
This function doesn't seem to be used anymore, please remove it or use it instead of the other function.